Gemfile

```diff
@@ -12,6 +12,7 @@ group :test do
   gem "omniauth-openid", "~> 1.0.1"
   gem "webrat", "0.7.3", require: false
   gem "mocha", "~> 1.1", require: false
+  gem 'test_after_commit', require: false
```

Comment:
Seems I have tried this before, didn't manage to succeed. I will look into it again and get back to you. Thanks !